src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/CSharpSyntaxGenerator.csproj

```diff
@@ -5,7 +5,8 @@
     <RootNamespace>CSharpSyntaxGenerator</RootNamespace>
     <!-- We build this against net7.0 to build the console app version, and against netstandard2.0 for
          the Source Generator version. -->
-    <TargetFrameworks>net7.0;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(DotNetBuildFromSource)' != 'true'">net7.0;$(TargetFrameworks)</TargetFrameworks>
```

Comment:
Is there a reason why we _need_ `net7.0` here? I think we could simplify this by just changing to 

```xml
<TargetFrameworks>net8.0;netstandard2.0</TargetFrameworks>
```

Reply:
I don't think so, this could very easily be 8.0.

Reply:
This can be simplified as you suggest, but needed to make `generate-compiler-code.ps1` more complex...